src/Enum/JitsiEnum.php

```diff
@@ -72,4 +72,6 @@ class JitsiEnum extends Enum
         'ā' => 'a', 'č' => 'c', 'ē' => 'e', 'ģ' => 'g', 'ī' => 'i', 'ķ' => 'k', 'ļ' => 'l', 'ņ' => 'n',
         'š' => 's', 'ū' => 'u', 'ž' => 'z',
     ];
+
+    public const DEFAULT_MODE = 'jitsi';
```

Comment:
imho settings those kind of flags is a mistake 

we have 4 options /modes: 

1. module is disabled 
2. module is enabled but no config data is put -> we're using public j instance https://meet.jit.si/
3. module is enabled  and `host` `app_id` & `app_secret` are set -> we're using self hosted jisty 
4. module is enabled  and `jaas_package_status` `jaas_host`, `aud`, `iss`, `kid`  & `private_key` are set -> we're Jitsy as a Service (jaas)